.gitignore

```diff
@@ -0,0 +1,12 @@
+/**/__pycache__/
+/**/output.sas
+/**/sas_plan*
```

Comment:
If we want to match the files/directories anywhere in our repositories, I would skip the leading '/'. Less is more.
**/foo matches any file/directory foo in our repository. 
/**/foo matches any file/directory found when starting to look at the directory root (is the same)

(https://git-scm.com/docs/gitignore)